src/test/resources/testscripts/NextflowRnaseq.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+## This script runs a demo Nextflow workflow on the GLS Pipelines API. The inline comments are talking notes for a demo.
+
+# Currently, we're connected to the Terra development server, and there is no current workspace defined.
+
+terra status
+
+# Now we can create a workspace. This will prompt a login. The scary-looking login screen is temporary, until we get our CLI app approved by Google.
+
+terra workspace create
+
+# [While that's running] This calls WSM to get a new Terra workspace. WSM in turn calls RBS to get a buffered GCP project, because creating a new project from scratch takes a while. The next part of this demo will run Nextflow against this GCP project. There were some short-term modifications we had to make to the buffered project configuration to support Nextflow.
+
+# [If that's still running] Nextflow expects the default VPC network and default Compute Engine service account to be defined. The "right" solution here is to modify the Nextflow codebase to allow specifying a non-default VPC network and Compute Engine service account. But for the alpha demo/release, it was more expedient to just modify the project configuration, and plan to make Nextflow code changes down the line.
+
+# Now the workspace was created. Check the status again to see the workspace id and backing GCP project.
+
+terra status
+
+# Nextflow requires a bucket to store temporary files. We can create a controlled resource for this, which means a cloud resource (a bucket in this case) within the backing GCP project.
+
+resourceName=terraclitesting$RANDOM
+terra resources create gcs-bucket --name=$resourceName --bucket-name=$resourceName
```

Comment:
bucket names and resource names have different practical restrictions. Might be good to demonstrate that there's a difference, and that every resource gets a name visible in Terra.

Reply:
Yes, good point. Also, while I was working on the GH action changes I realized `$RANDOM` doesn't work on the GH action runner for some reason. So I overhauled both things in the follow-on PR.